ui/src/app/shared/components/orb/sink-display/sink-display.component.ts

```diff
@@ -21,8 +21,9 @@ export class SinkDisplayComponent implements OnInit {
   ngOnInit(): void {
   }
 
-  onRemoveSink(sinkID) {
-    this.sinksChange.emit(this.sinks.filter(sink => sink.id !== sinkID));
+  onRemoveSink(sink) {
+    this.sinks.splice(this.sinks.indexOf(sink), 1);
+    this.sinksChange.emit(this.sinks);
```

Comment:
When a sink is removed, it should be shown again as a choice



![remove_sink](https://user-images.githubusercontent.com/78241475/165314387-48840f93-4057-41f9-ab79-b2ef791c7fec.gif)